classification/train.py

```diff
@@ -21,24 +21,31 @@
 from torchmetrics.classification import MultilabelAveragePrecision
 
 import torch
+from torch.utils.data import DataLoader
 import torch.nn.functional as F
 from torch.optim import Adam
-from torch.amp import autocast
+from torch.amp.autocast_mode import autocast
 import numpy as np
-from dataset import PyhaDF_Dataset, get_datasets
-from model import TimmModel, EarlyStopper
-from utils import set_seed, print_verbose
-import config
-from augmentations import SyntheticNoise
 from tqdm import tqdm
 import wandb
 
+from utils import set_seed, print_verbose
+
+
+
+
+from augmentations import SyntheticNoise
+import config
+from dataset import  get_datasets
+
+from models.timm_model import TimmModel
+from models.early_stopper import EarlyStopper
 
 
 tqdm.pandas()
 time_now  = datetime.datetime.now().strftime('%Y%m%d-%H%M') 
 device = 'cuda' if torch.cuda.is_available() else 'cpu'
-wandb_run = None
+#wandb_run = wandb.run
```

Comment:
can we remove this entirely

Reply:
This got fixed in the base branch so I'll just fix it in a merge commit